### ISSUE TITLE
Issue with lookup table in plot_easy_example

### ISSUE THREAD

**Issue by RianneSchouten** [RianneSchouten/pyampute#27]
The quick example shows some warnings because the table cannot be located. 
See: https://rianneschouten.github.io/pyampute/build/html/auto_examples/plot_easy_example.html.

does this have to do with the location of the example file?

**Comment by RianneSchouten** [comment]
Actually, the issue also occurs in general. I have installed with pip install pyampute, and then run in the console and it happens. 

@davzaman if you have time, can you have a look?

Python 3.8.1 (tags/v3.8.1:1b293b6, Dec 18 2019, 23:11:46) [MSC v.1916 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import pyampute.ampute as amp
>>> import numpy as np
>>> n = 1000
>>> m = 10
>>> X_compl = np.random.randn(n,m)
>>> ma = amp.MultivariateAmputation()
>>> ma.fit_transform(X_compl)
2022-02-11 20:38:40,209 [WARNING] Failed to load lookup table for a prespecified score to probability function. It is possible /data/C:\WINDOWS\system32\data\shift_lookup.csv.csv is missing, in the wrong location, or corrupted. Try rerunning /amputation/scripts.py to regenerate the lookup table.

**Comment by davzaman** [comment]
` /data/C:\WINDOWS\system32\data\shift_lookup.csv.csv` this is malformed, it may have to do with how I'm putting together the filename for opening, I will take a look.

**Comment by davzaman** [comment]
fixed with 990c0b7

**Comment by RianneSchouten** [comment]
@davzaman Thank you for having a look at this issue. 

I am re-opening the issue, because something is not clear to me. 
When I now run the sphinx documentation (from the docs folder, using make html), the quick example still shows the warning, but the other examples do not (see https://rianneschouten.github.io/pyampute/build/html/auto_examples/plot_easy_example.html).

I do not understand this, since all examples live in the same folder (examples/). 

If you have time, can you have a look?

**Comment by davzaman** [comment]
You're getting the same error about not being able to find the table? I just did a fresh pull of the repo and installed everything in a new env and ran it and it seems to work fine. This is my output:
```
Running Sphinx v4.4.0
WARNING: html_static_path entry '_static' does not exist
loading pickled environment... failed
failed: source directory has changed
[autosummary] generating autosummary for: about.rst, auto_examples/index.rst, auto_examples/plot_custom_pipeline.rst, auto_examples/plot_custom_probability_function.rst, auto_examples/plot_easy_example.rst, auto_examples/plot_simulation_pipeline.rst, auto_examples/sg_execution_times.rst, blogposts.rst, contribution.rst, examples.rst, index.rst, installation.rst, mapping.rst, pyampute.ampute.rst, pyampute.exploration.rst, pyampute.rst
generating gallery...
Using Sphinx-Gallery to convert rst text blocks to markdown for .ipynb files.
generating gallery for auto_examples... [100%] plot_custom_pipeline.py                       
computation time summary:
    - ../../examples/plot_custom_pipeline.py:               0.00 sec   0.0 MB
    - ../../examples/plot_custom_probability_function.py:   0.00 sec   0.0 MB
    - ../../examples/plot_easy_example.py:                  0.00 sec   0.0 MB
    - ../../examples/plot_simulation_pipeline.py:           0.00 sec   0.0 MB
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 16 source files that are out of date
updating environment: [new config] 16 added, 0 changed, 0 removed
reading sources... [100%] pyampute.exploration                                               
/home/dav/research/pyampute/docs/source/mapping.rst:38: WARNING: image file not readable: figures/mdpatterns_nhanes2.png
/home/dav/research/pyampute/docs/source/mapping.rst:: WARNING: image file not readable: figures/wss_plots.png
/home/dav/research/pyampute/pyampute/ampute.py:docstring of pyampute.ampute.MultivariateAmputation.DEFAULTS:1: WARNING: duplicate object description of pyampute.ampute.MultivariateAmputation.DEFAULTS, other instance in pyampute.ampute, use :noindex: for one of them
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] pyampute.exploration                                                
generating indices... genindex py-modindex done
writing additional pages... search done
copying images... [ 40%] auto_examples/images/thumb/sphx_glr_plot_simulation_pipeline_thumb.pcopying images... [ 60%] auto_examples/images/thumb/sphx_glr_plot_custom_probability_functioncopying images... [100%] figures/types.png                                                   
copying downloadable files... [100%] figures/scheme.pdf                                      
copying static files... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build succeeded, 4 warnings.

The HTML pages are in build/html.

Sphinx-Gallery successfully executed 0 out of 0 files subselected by:

    gallery_conf["filename_pattern"] = '/plot'
    gallery_conf["ignore_pattern"]   = '__init__\\.py'

after excluding 4 files that had previously been run (based on MD5).

embedding documentation hyperlinks...
embedding documentation hyperlinks for auto_examples... [ 83%] plot_custom_probability_functiembedding documentation hyperlinks for auto_examples... [100%] index.html                  
```

**Comment by RianneSchouten** [comment]
Hi @davzaman 

I am actually still getting this warning, even when running a simple example.
Can you once more have a look?

```
import numpy as np
import pandas as pd
from pyampute import *

n = 10000
X = np.random.randn(n, 3)
X = pd.DataFrame(X, columns = ['A','B','C'])
print(X.info())

ma = MultivariateAmputation()
X_amp = ma.fit_transform(X)
print(X_amp.info())
```

**Comment by davzaman** [comment]
I see what you're saying. My guess is that it has to do with how building the docs sets the path, and I was right. I just had to move/copy the data/lookup.csv file into the examples folder. Fixed with 2aa352a

**Comment by Theusii** [comment]
Hello! Im quite new here but very fond of the package you and your colleagues made! Im writing my thesis about the how LLM can revolutionize the field  of imputation and i am using your package to ampute my data in Jupyter Notebooks. I was having the same trouble, always getting the error "Failed to load lookup table for a prespecified score to probability function. It is possible data/shift_lookup.csv is missing, in the wrong location, or corrupted. Try rerunning scripts/generate_shift_lookup_table.py to regenerate the lookup table." I did ask Chat GPT but did find no help and i tried to explain to it everything you wrote here, since i dont really understand what exactly you did to fix the error, could you maybe explain again ? i would so thankful for that. Ive been trying to fix this for over 2 days now.
